package.json

```diff
@@ -12,7 +12,7 @@
     "publish-package": "npm run build && npm publish",
     "report-coverage": "npm run test-cover && ./node_modules/coveralls/bin/coveralls.js < ./coverage/lcov.info",
     "tdd": "mocha -w --compilers js:babel/register 'test/**/*_test.js'",
-    "test": "npm run lint && npm run test-nocover",
+    "test": "npm run lint && npm run report-coverage",
```

Comment:
> Running local tests with npm test will now make a call to coveralls ?

You're right we shouldn't. Maybe it's okay to run this twice after all. Closing.